modules/apps/questions/questions-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -13,6 +13,14 @@
 
 	<%
 	QuestionsConfiguration questionsConfiguration = ConfigurationProviderUtil.getPortletInstanceConfiguration(QuestionsConfiguration.class, themeDisplay);
+
+	long categoryId = 0;
```

Comment:
You should use existing constants when possible (see `MBCategoryConstants`).

Reply:
Done. See last commit [574c892](https://github.com/liferay-content-management/liferay-portal/pull/5927/commits/574c892318435a0934d4a9a824b8ecb6d522ebb8)